docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -0,0 +1,326 @@
+= Configuring the JDBC Driver
+:url-dbeaver-jdbc: https://dbeaver.com/docs/wiki/Database-drivers/
+:description: To start using the Hazelcast JDBC driver, you need to load the JDBC driver and establish a connection to your Hazelcast cluster. To do this, some configuration is required.
+
+{description}
+
+You'll need to add the following details to your client application or tool:
+
+- <<jdbc-driver-class-name, JDBC driver class name>> 
+- <<connection-url, Connection string or URL>>
+
+NOTE: It is also possible to use xref:hazelcast:configuration:configuring-declaratively.adoc[configuration files] to set or xref:hazelcast:configuration:configuring-declaratively.adoc#overriding-configuration-with-system-properties-and-environment-variables[override the configuration] of your Java client application.
+
+[[jdbc-driver-class-name]]
+== Adding the JDBC Driver Class Name
+
+To load the JDBC driver, either add the class name `com.hazelcast.jdbc.Driver` to your client application or preferred tool.
+
+For a client application written in Spring Boot, you might add the following to your `application.properties`.
+
+[source,bash]
+----
+spring.datasource.driver-class-name=com.hazelcast.jdbc.Driver
+----
+
+Or, if you're using a tool like DBeaver, you'll add the class name to your link:{url-dbeaver-jdbac}[driver configuration].
+
+[[connection-url]]
+== Building a Connection URL
+
+The JDBC connection URL for a Hazelcast cluster has the following format:
```

Comment:
```suggestion
After you've loaded the JDBC driver, the next step is to build a connection URL so that the JDBC driver can connect to your Hazelcast Cluster.

The JDBC connection URL for a Hazelcast cluster has the following format:
```